client/state/data-layer/wpcom/jetpack/settings/index.js

```diff
@@ -88,7 +88,9 @@ export const announceRequestFailure = ( { dispatch, getState }, { siteId } ) =>
 export const saveJetpackOnboardingSettings = ( { dispatch }, action ) => {
 	const { settings, siteId } = action;
 
-	dispatch( updateJetpackOnboardingSettings( siteId, action.settings ) );
+	// We don't want JP Onboarding specific things in our JP Settings Redux state
```

Comment:
Let's change JP to `Jetpack`, and generally I'd change `JP Onboarding specific things` to `Jetpack Onboarding credentials`